packages/angular_devkit/build_angular/src/builders/browser-esbuild/index.ts

```diff
@@ -51,4 +63,39 @@ function normalizeOptions(options: BrowserBuilderOptions): ApplicationBuilderOpt
   };
 }
 
+async function writeResultFiles(
```

Comment:
Can you add a comment for this function about how this is here for compatibility with the Webpack output structure?